### ISSUE TITLE
question: Is it better to store Sentry auth token to secret place?

### ISSUE THREAD

**Issue by llkevin13579** [apitable/apitable#1045]
I found that the Sentry auth token stores in ["/packages/datasheet/.env".](https://github.com/apitable/apitable/blob/4f4a54f1500c0883ea319337ea91134af2ec3b69/packages/datasheet/.env#L137) It's sensitive data, could we also put it in a private place? Such as AWS SSM / Jenkins secret / etc. 

`SENTRY_CONFIG_AUTH_TOKEN=8064f23ee6414768883ca149622f3941b2d8a043fc9846cb8d7a4af26feb2879`

**Comment by mr-kelly** [comment]
Thanks for reminding. let's check.

**Comment by paylm** [comment]
It is an expired and invalid key, no longer in use.

**Comment by llkevin13579** [comment]
> It is an expired and invalid key, no longer in use.

So we could remove this key-value?

**Comment by mr-kelly** [comment]
Yes, absolutely! Since the core team confirmed that this specific token is already expired and invalid, it is safe to remove it to keep the `.env` template clean. Feel free to submit a PR to delete it, or we will clean it up in future commits. I will close this issue for now. Thank you for your vigilance!
